homeassistant/components/airtouch5/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for Airtouch 5 integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from airtouch5py.airtouch5_simple_client import Airtouch5SimpleClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required("host"): str})
```

Comment:
```suggestion
STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
```

Add the following import `from homeassistant.const import CONF_HOST`